packages/react-components/src/components/styles/TogetherMode.styles.ts

```diff
@@ -0,0 +1,186 @@
+  seatingPosition: VideoGalleryTogetherModeSeatingInfo
+): TogetherModeParticipantSeatPosition {
+  return {
+    width: _pxToRem(seatingPosition.width),
+    height: _pxToRem(seatingPosition.height),
+    left: _pxToRem(seatingPosition.left),
+    top: _pxToRem(seatingPosition.top)
+  };
+}
+
+/* @conditional-compile-remove(together-mode) */
+/**
+ * Return a style bucket based on the number of active sprites.
+ * For example, the first three reactions should appear at maximum
+ * height, width, and opacity.
+ * @private
+ */
+export function setTogetherModeSeatPositionStyle(
+  seatingPosition: VideoGalleryTogetherModeSeatingInfo
+): TogetherModeSeatStyle {
+  return {
+    seatPosition: setParticipantSeatingPosition(seatingPosition)
+  };
+}
+
+/* @conditional-compile-remove(together-mode) */
+/**
+ * Generates the overlay style for a participant in Together Mode.
+ *
+ * @param seatingPosition - The seating position information.
+ * @returns The style object for the participant overlay.
+ */
+export function getTogetherModeParticipantOverlayStyle(
+  seatingPositionStyle: TogetherModeSeatStyle
+): React.CSSProperties {
+  return {
+    ...seatingPositionStyle.seatPosition,
+    position: 'absolute'
+  };
+}
+
+// Function to map a value from one range to another
+const mapRange = (value: number, inMin: number, inMax: number, outMin: number, outMax: number): number => {
+  return outMin + ((value - inMin) * (outMax - outMin)) / (inMax - inMin);
+};
+
+/**
+ * Calculate the scaled size based on width and height.
+ *
+ * @param width - The width of the element.
+ * @param height - The height of the element.
+ * @returns The scaled size.
+ */
+export const calculateScaledSize = (width: number, height: number): number => {
+  const maxSize = 600;
+  const minSize = 200;
+  const minScaledSize = 35;
+  const maxScaledSize = 70;
+
+  // Use width or height to determine scaling factor
+  const size = Math.min(width, height);
+
+  // Map the size to the desired range
+  return mapRange(size, minSize, maxSize, minScaledSize, maxScaledSize);
+};
+
+/**
+ * Get the root participant status div style.
+ *
+ * @returns The style object for the root participant status div.
+ */
+export const getRootParticipantStatusDivStatus = (): React.CSSProperties => {
+  return {
+    width: '100%',
+    height: '100%',
+    position: 'absolute',
+    top: 0,
+    left: 0
+  };
+};
+
+/* @conditional-compile-remove(together-mode) */
+/**
+ * @private
+ */
+export const togetherModeStreamRootStyle: IStackStyles = {
+  root: {
+    width: '100%',
+    height: '100%',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    position: 'absolute', // Prevents sliding due to resizing
+    top: 0, // Anchors the div at the top
+    left: 0 // Anchors the div at the left
+  }
+};
+
+/* @conditional-compile-remove(together-mode) */
+/**
+ * @private
+ */
+export const togetherModeIconStyle = (): React.CSSProperties => {
+  return {
+    width: '20px',
```

Comment:
lets avoid px, you should be able to set as rem. This is for Accessibility reasons as if you are just setting it with px then when the user zooms the page the icon wont change size. this is because rem follows the defined font size for the page and the font size changes under zoom
![image](https://github.com/user-attachments/assets/eb44b94b-19c7-4404-aa7e-d41b944c8eed)
https://developer.mozilla.org/en-US/docs/Learn_web_development/Core/Styling_basics/Values_and_units